ArduCopter/RC_Channel.cpp

```diff
@@ -105,6 +105,12 @@ void RC_Channel_Copter::init_aux_function(const aux_func_t ch_option, const aux_
     case AUX_FUNC::ZIGZAG:
     case AUX_FUNC::ZIGZAG_SaveWP:
         break;
+    case AUX_FUNC::STABILIZE:
```

Comment:
Should these go above the break?!

Reply:
I think the break is for ZIGZAG_SaveWP, maybe?

Reply:
Ah.  I think you might like to read up on the way switch statements work in C/C++ :-)

Please move the break to underneath your lines.

Reply:
My bad. Fixed